post-install.yml

```diff
@@ -0,0 +1,146 @@
+- hosts: "{{ hosts_prefix }}-tsuru-api*[0]"
+  sudo: no
+  tasks:
+    - name: Tsuru target-list
+      shell: >
+        tsuru target-list
+      register: tsuru_targets
+    - name: Tsuru add target.
+      shell: >
+        tsuru target-add {{ deploy_env }}_{{ domain_name }} http://127.0.0.1:8080
+      when: "not '{{ deploy_env }}_{{ domain_name }}' in tsuru_targets.stdout"
+    - name: Tsuru set target.
+      shell: >
+        tsuru target-set {{ deploy_env }}_{{ domain_name }}
+    - name: Tsuru login
+      shell: >
+        echo {{ admin_password }} | tsuru login {{ admin_user }}
+
+    - name: Deploy insecure-deployer.pub key
+      copy:
+        src: files/insecure-deployer.pub
```

Comment:
> Generating new key every time would mean we/jenkins would have to deploy a new key for every new environment...

Is that a bad thing? We could use our personal keys and Jenkins could use its own (which would be generated on and never leave the Jenkins server).

I'd like to stop using the `insecure-deployer` keypair because we've shared it amongst many users and have no way of individually revoking them.